app/models/contact.rb

```diff
@@ -0,0 +1,10 @@
+class Contact < ApplicationRecord
+  MAX_TEXT_LENGTH = 60
+  validates :name,
```

Comment:
There is better to use separate length validation because mobile number can not be 60 characters, and address can be more than 60